packages/core-components/src/lib/atoms/inputs/date-range/_DateRange.svelte

```diff
@@ -0,0 +1,159 @@
+<script lang="ts">
+	import { Calendar as CalendarIcon } from 'radix-icons-svelte';
+	import type { DateRange } from 'bits-ui';
+	import {
+		CalendarDate,
+		DateFormatter,
+		getLocalTimeZone,
+		startOfMonth,
+		endOfMonth
+	} from '@internationalized/date';
+	import { cn } from '$lib/utils.js';
+	import { Button } from '$lib/atoms/shadcn/button/index.js';
+	import { RangeCalendar } from '$lib/atoms/shadcn/range-calendar/index.js';
+	import * as Select from '$lib/atoms/shadcn/select/index.js';
+	import * as Popover from '$lib/atoms/shadcn/popover/index.js';
+
+	function YYYYMMDDToCalendar(yyyymmdd: string) {
+		const pieces = yyyymmdd.split('-');
+		return new CalendarDate(Number(pieces[0]), Number(pieces[1]), Number(pieces[2]));
+	}
+
+	const dfMedium = new DateFormatter('en-US', {
+		dateStyle: 'medium'
+	});
+
+	const dfShort = new DateFormatter('en-US', {
+		dateStyle: 'short'
+	});
+
+	export let selectedDateRange: DateRange | undefined;
+	export let start: string;
+	export let end: string;
+
+	$: calendarStart = YYYYMMDDToCalendar(start);
```

Comment:
hard reload managed to get be Jan 1st 1970 again
![CleanShot 2024-02-06 at 18 26 22@2x](https://github.com/evidence-dev/evidence/assets/58074498/d99afd0f-d495-4971-b9c6-00c70dc9faa1)